roles/provision-metrics-apb/tasks/provision-app-metrics-ephemeral.yml

```diff
@@ -90,18 +91,38 @@
         port: 443
         target_port: '{{ api_server_port }}'
 
-- name: Create api server route
+- name: Create api server https route
   openshift_v1_route:
     name: aerogear-app-metrics
     namespace: '{{ namespace }}'
     labels:
-      app: aerogear-app-metrics
+      app: metrics
       service: aerogear-app-metrics
     to_name: aerogear-app-metrics
     spec_port_target_port: web
+    tls_termination: edge
+    tls_insecure_edge_termination_policy: Redirect
   register: app_metrics_route
 
-    
+# Create the configmap used for the client config
+- name: Create metrics configmap template
+  template:
+    src: configmap.yml.j2
+    dest: /tmp/configmap.yaml
+  when: _apb_service_instance_id is defined
+
+- name: Create metrics test configmap template
+  template:
+    src: configmap_test.yml.j2
+    dest: /tmp/configmap.yaml
+  when: _apb_service_instance_id is not defined
+
+- name: Create metrics public client config map
+  shell: oc create -f /tmp/configmap.yaml -n {{ namespace }}
+
+- name: delete configmap template file
+  file: path=/tmp/configmap.yaml state=absent
+
```

Comment:
@psturc what's this block doing?

Reply:
Ok, got it. we were missing the config map on ephemeral installations. Thanks for fixing this.

Reply:
This file is just a copy-pasta of [provision-app-metrics.yml file](https://github.com/aerogearcatalog/metrics-apb/blob/master/roles/provision-metrics-apb/tasks/provision-app-metrics.yml) except PV and PVC configuration. It was needed because the tests were failing on Wendy if Postgres was deployed with persistent volume attached to it.
I've only updated it, because some changes from `provision-app-metrics.yml` were apparently not applied to this file.

Reply:
But when I'm thinking about it now, we can probably remove the whole file, because testing on ci-rhos (as proposed in this PR: https://github.com/aerogearcatalog/metrics-apb/pull/64) does not have this issue with PVs anymore.
I'll update this PR once https://github.com/aerogearcatalog/metrics-apb/pull/64 is merged